ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableQueries.scala

```diff
@@ -6,4 +6,6 @@ package com.daml.platform.store.dao.events
 private[events] object EventsTableQueries {
   def format(ps: Set[Party]): String =
     ps.view.map(p => s"'$p'").mkString(",")
+  def format(ps: List[Party]): String =
+    ps.mkString(",")
```

Comment:
Why do we format `List[Party]` different than `Set[Party]`?

Reply:
Bad copy-paste, thanks for noticing!